frontend/src/components/associations/events/List.tsx

```diff
@@ -10,41 +10,51 @@ import { PageTitle } from "../../utils/PageTitle";
 import { Pagination } from "../../utils/Pagination";
 import { Association } from "../../../models/associations/association";
 import { EventCard } from "./EventCard";
+import { UserContext } from "../../../services/authService";
 
 export const AssociationEventList = ({
     association,
 }: {
     association: Association;
-}) => (
-    <Pagination
-        apiKey={["events.get", { associationId: association.id }]}
-        apiMethod={api.events.list}
-        loadingElement={LoadingAssociation}
-        render={(events, paginationControl) => (
-            <Container className="mt-5">
-                {association.myRole.permissions.includes("media") && (
-                    <Link
-                        to={`/associations/${association.id}/evenements/nouveau`}
-                        className={"btn btn-success float-right mt-5"}
-                    >
-                        <i className="fe fe-upload" />
-                        Ajouter un évenement
-                    </Link>
-                )}
+}) => {
+    const user = useContext(UserContext);
 
-                <PageTitle className="mt-6">Événements</PageTitle>
+    return (
+        <Pagination
+            apiKey={["events.list", { associationId: association.id }]}
+            apiMethod={api.events.list}
+            loadingElement={LoadingAssociation}
+            render={(events, paginationControl) => (
+                <Container className="mt-5">
+                    {association.myRole.permissions.includes("media") && (
+                        <Link
+                            to={`/associations/${association.id}/evenements/nouveau`}
+                            className={"btn btn-success float-right mt-5"}
+                        >
+                            <i className="fe fe-upload" />
+                            Ajouter un évenement
```

Comment:
<img width="467" alt="image" src="https://user-images.githubusercontent.com/4300678/82676229-553e2000-9c46-11ea-9c2c-f6eed3cbb98b.png">

Button is a bit sticky